scripts/utils.py

```diff
@@ -0,0 +1,33 @@
+def find_features(string_f, labels, unique=False):
+    """
+    :param complement: if true, return indexes not in string_f
+    :param unique: to take only unique fields
+    :param string_f: feature string
+    :param labels: first row of the data
+    :return: indexes of feature ti be shown
+    """
+
+    if len(string_f) is 0:
```

Comment:
As this comparison will work as you expect it to work, it 's better to use == because everybody uses == in such cases.